odoo_test_helper/fake_model_loader.py

```diff
@@ -147,9 +147,8 @@ def restore_registry(self):
 
         self._clean_module_to_model()
 
-        # reload is need to resetup correctly the field on the record
+        # reload is need to reset correctly the field on the record
         with mock.patch.object(self.env.cr, "commit"):
-            self.env.registry.model_cache.clear()
```

Comment:
We should only remove this for version odoo15 (so we should add a condition if odoo.version < 15.0)

Reply:
updated